autoreject/autoreject.py

```diff
@@ -1218,26 +1218,51 @@ def __init__(self, bad_epochs, labels, ch_names):
         assert len(bad_epochs) == labels.shape[0]
         assert len(ch_names) == labels.shape[1]
 
-    def plot(self):
-        """Plot."""
+    def plot(self, horizontal=False, show=True):
```

Comment:
I would call this parameter `orientation` which takes two params `'vertical'` and `'horizontal'`. To mimic matplotlib API as much as possible

Reply:
done!